app/routes/_marketing+/index.tsx

```diff
@@ -133,6 +133,11 @@ export default function Index() {
 								alt: 'ESLint',
 								href: 'https://eslint.org',
 							},
+							{
+								src: 'https://svgur.com/i/t9k.svg',
```

Comment:
Maybe we want to host his somewhere else?

Reply:
I just drag-and-drop the image into a github textbox and copy the URL that it gives me.